main/src/cgeo/geocaching/maps/CGeoMap.java

```diff
@@ -456,7 +453,8 @@ protected void initializeMap(final ArrayList<Location> trailHistory) {
             followMyLocation = false;   // do not center on GPS position, even if in LIVE mode
         }
 
-        overlayPositionAndScale = mapView.createAddPositionAndScaleOverlay(getIntentCoords(), mapOptions.geocode);
+        Viewport viewport = getIntentViewport();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'viewport' should be declared final.](https://app.codacy.com/manual/cgeo/cgeo/pullRequest?prid=4742102)